### ISSUE TITLE
create_plan initial_price check bug

### ISSUE THREAD

**Issue by smtydn** [paddle-python/paddle-client#2]
Hi. I realized that there is a bug in client's initial_price existence logic. I would want to create a PR for it but I don't have any access to a Paddle account. I would be happy to contribute if any Paddle account is given for testing, otherwise I'm okay to see that fixed. Thanks for your time.

```
    if main_currency_code == 'USD' and initial_price_usd is None:
        raise ValueError('main_currency_code is USD so initial_price_usd must be set')  # NOQA: E501
    if main_currency_code == 'GBP' and initial_price_gbp is None:
        raise ValueError('main_currency_code is USD so initial_price_gbp must be set')  # NOQA: E501
    if main_currency_code == 'USD' and initial_price_eur is None:
        raise ValueError('main_currency_code is USD so initial_price_eur must be set')  # NOQA: E501
    if main_currency_code == 'USD' and recurring_price_usd is None:
        raise ValueError('main_currency_code is USD so recurring_price_usd must be set')  # NOQA: E501
    if main_currency_code == 'GBP' and recurring_price_gbp is None:
        raise ValueError('main_currency_code is USD so recurring_price_gbp must be set')  # NOQA: E501
    if main_currency_code == 'USD' and recurring_price_eur is None:
        raise ValueError('main_currency_code is USD so recurring_price_eur must be set')  # NOQA: E501
```

**Comment by pyepye** [comment]
Hi @smtydn 

I've made a change to fix a bug with the `EUR` pricing - https://github.com/paddle-python/paddle-client/pull/3
Hopefully this is the bug you were referencing, if not please let us know.

**Comment by smtydn** [comment]
Thanks @pyepye. That was exactly what I was looking for.

**Comment by pyepye** [comment]
Great thanks for confirming @smtydn 

I've just merged and created version [`0.8.1`](https://github.com/paddle-python/paddle-client/releases/tag/0.8.1) and pushed to PyPi

The [paddle-client page on PyPI](https://pypi.org/project/paddle-client/) is showing 0.8.1 now but it usually takes a little time to propagate so pip install / upgrade might not reflect the change quite yet. It's normally pretty fast but can sometimes take an hour or 2.
